modules/extension/complex/src/main/java/org/geotools/data/complex/feature/type/ComplexTypeProxy.java

```diff
@@ -27,7 +27,7 @@
  * @author Gabriel Roldan (Axios Engineering)
  * @since 2.4
  */
```

Comment:
Would be nice to add meaningful docs to classes if they are becoming public

Reply:
They were previously relying on package visibility, and can't anymore.